src/app/shared/shared.module.ts

```diff
@@ -40,34 +33,38 @@ import { ImageModule, SpinnerModule } from 'ngx-prx-styleguide';
     WysiwygComponent
   ],
   exports: [
-    AdvancedConfirmDirective,
     AdvancedSectionComponent,
     CommonModule,
     FormsModule,
-    ButtonComponent,
-    CapitalizePipe,
     CopyInputDirective,
     DatepickerComponent,
+    DatepickerModule,
     DurationPipe,
-    FancyDurationComponent,
-    FancyFieldComponent,
+    FancyFormModule,
     FileSelectDirective,
     FileSizePipe,
     FocusDirective,
+    HeroModule,
+    ImageModule,
     ImageUploadComponent,
     TabComponent,
     TimeAgoPipe,
     TimepickerComponent,
     TextOverflowEllipsesComponent,
     TextOverflowFadeComponent,
     TextOverflowWebkitLineClampComponent,
+    SpinnerModule,
     WysiwygComponent
   ],
   imports: [
     CommonModule,
     FormsModule,
     RouterModule,
+    DatepickerModule,
```

Comment:
do these have to be in non-alpha order in order to maintain order of dependencies?